api/api.proto

```diff
@@ -932,6 +932,50 @@ service OpenStorageCluster {
         get: "/v1/clusters/current"
       };
     }
+    // Creates Pair with a remote cluster and returns details about the remote cluster
+    rpc CreatePair(ClusterPairCreateRequest)
```

Comment:
I suggest creating a single issue with the API proposal and then create a PR per API function.